cmd/autoremove.go

```diff
@@ -5,8 +5,8 @@ package cmd
 		Mirko Brombin <send@mirko.pm>
 		Pietro di Caprio <pietro@fabricators.ltd>
 	Copyright: 2022
-	Description: Apx is a wrapper around apt to make it works inside a container
-	from outside, directly on the host.
+	Description: Apx is a wrapper around apt to make it work inside a container
```

Comment:
Since apx is no more "just" a wrapper of Apt, we should change this sentence to something else, maybe a "a wrapper around multiple package managers.."